lang/fr.json

```diff
@@ -52,8 +52,8 @@
         "download": "Télécharger un Fichier de Sauvegarde"
       },
       "uiTheme": "Thème UI",
-      "toggleChat": "Toggle Chat",
-      "toggleNametags": "Toggle Nametags"
+      "toggleChat": "Afficher le Tchat",
```

Comment:
Is 'tchat' a typo? Just checking.

Reply:
Not a typo; it's a French equivalent to "chat".